src/Microsoft.Extensions.ProjectModel.Sources/MsBuild/MsBuildProjectContextBuilder.cs

```diff
@@ -19,11 +19,11 @@ namespace Microsoft.Extensions.ProjectModel
 {
     internal class MsBuildProjectContextBuilder
     {
-        private string _configuration;
-        private IFileInfo _fileInfo;
-        private string[] _buildTargets;
-        private Dictionary<string, string> _globalProperties = new Dictionary<string, string>();
-        private bool _explicitMsBuild;
+        protected string _configuration;
```

Comment:
Nit: always use properties with PascalCase names instead of protected fields 
`protected string Configuration { get; set; }`